constants/appLinks.ts

```diff
@@ -77,6 +79,16 @@ const TEXTBOOK_DEMO_START_LEARNING: GeneralLink = {
   }
 }
 
+const textbookDemoIntroductionCourseStartLearningUrl = 'https://project-platypus-default.apis-dev-5dd5718798d097eccc65fac4e78a33ce-0000.us-south.containers.appdomain.cloud/course/learning-states-and-qubits/introduction'
```

Comment:
We should move this url to a better place to be close to where it's used, ideas? In addition, you can change the name to `introductionLearningPathIntroductionSectionURL` or something similar to be more clear

Reply:
Lots of ideas 😄  But as discussed, I thought we were going to agree on the proper way to handle this going forward with https://github.com/Qiskit/qiskit.org/issues/1566.